aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -814,11 +831,11 @@ void raw_cudnn_convolution_forward_out_32bit(
 
   ConvolutionArgs args{ input, output, weight };
   args.handle = getCudnnHandle();
-  setConvolutionParams(&args.params, input, weight, padding, stride, dilation, groups, deterministic);
+  setConvolutionParams(&args.params, input, weight, padding, stride, dilation, groups, deterministic, at::globalContext().allowTF32CuDNN());
```

Comment:
Should this be populated like benchmark and deterministic? See

https://github.com/pytorch/pytorch/blob/0d9e3b48c4ef54a7b5395c501ee370a8eba34bad/aten/src/ATen/native/Convolution.cpp#L579

Reply:
This would require a change of the signature of functions like `cudnn_convolution_forward`, therefore BC-breaking

Reply:
But you've already changed cudnn_convolution signature when you removed bias, and that was ok?

Reply:
@ngimel Yes, and the solution is to rename the old signature `*.deprecated`. Working on it now, but the size of this PR doubles because the signature of `_convolution` is baked deep inside JIT.